### PR TITLE
Indonesian Language Repair Update strings.xml

### PR DESCRIPTION
**Pull request by TakaeSlept**
Wrong Grammar And Typos

Nebulous ID: 17200046

### CODE REVIEW

**Comment by TakaeSlept** [comment]
Signed off by accident, when i commit it gets signed off automatically

File: res/values-in/strings.xml [hunk 1]
```diff
@@ -53,8 +53,8 @@
     <string name="SERVER">"SERVER"</string>
     <string name="Joining_game_in_">"Bergabung permainan di "</string>
     <string name="Game_Full">"Permainan Penuh"</string>
-    <string name="US_East">"Amerika Tengah"</string>
-    <string name="US_West">"Amerika Utara"</string>
+    <string name="US_East">"Amerika Utara"</string>
+    <string name="US_West">"Amerika Barat"</string>
```
Comment (Jeiii20) [inline res/values-in/strings.xml:57]: Wrong changes.

`US_East`: It should keep as "Amerika Tengah".
`US_West`: It should keep as "Amerika Utara.

You literally inverted and returned the old names. Those should not base off on the string name (in blue colour). Please return them as they were.

File: res/values-in/strings.xml [hunk 2]
```diff
@@ -78,17 +78,17 @@
...
-    <string name="Not_signed_in_">"Tidak Di-Login!"</string>
+    <string name="Signed_in_as_">"Login sebagai:"</string>
+    <string name="Not_signed_in_">"Tidak Ter-Login!"</string>
     <string name="SET_NAME">"SET NAMA"</string>
-    <string name="SIGN_OUT">"LOGOUT"</string>
+    <string name="SIGN_OUT">"LOG-OUT"</string>
     <string name="ACHIEVEMENTS">"PRESTASI"</string>
     <string name="STATISTICS">"STATISTIK"</string>
-    <string name="XP_">"XP:"</string>
+    <string name="XP_">"Pengalaman:"</string>
```
Comment (Jeiii20) [inline res/values-in/strings.xml:87]: "XP" is used everywhere. Should not be changed.

File: res/values-in/strings.xml [hunk 3]
```diff
@@ -123,7 +123,7 @@
     <string name="TEAMS_TIME">"WAKTU TIM"</string>
     <string name="CTF">"CTF"</string>
     <string name="SURVIVAL">"BERTAHAN HIDUP"</string>
-    <string name="Clan_XP">"XP Klan"</string>
+    <string name="Clan_XP">"Pengalaman Klan"</string>
```
Comment (Jeiii20) [inline res/values-in/strings.xml:126]: Same issue as line 87.

**Review by Jeiii20 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**